benchmarks/viscosity_grooves/viscosity_grooves.cc

```diff
@@ -0,0 +1,496 @@
+         */
+        virtual
+        std::pair<std::string,std::string>
+        execute (TableHandler &statistics);
+    };
+
+    template <int dim>
+    std::pair<std::string,std::string>
+    ViscosityGroovesPostprocessor<dim>::execute (TableHandler &)
+    {
+      std::shared_ptr<Function<dim> > ref_func;
+      {
+
+        ref_func.reset (new AnalyticSolutions::FunctionViscosityGrooves<dim>(this->get_geometry_model()));
+      }
+
+      const QGauss<dim> quadrature_formula (this->introspection().polynomial_degree.velocities+2);
+
+      Vector<float> cellwise_errors_u (this->get_triangulation().n_active_cells());
+      Vector<float> cellwise_errors_p (this->get_triangulation().n_active_cells());
+      Vector<float> cellwise_errors_ul2 (this->get_triangulation().n_active_cells());
+      Vector<float> cellwise_errors_pl2 (this->get_triangulation().n_active_cells());
+
+      ComponentSelectFunction<dim> comp_u(std::pair<unsigned int, unsigned int>(0,dim),
+                                          dim+2);
+      ComponentSelectFunction<dim> comp_p(dim, dim+2);
+
+      VectorTools::integrate_difference (this->get_mapping(),this->get_dof_handler(),
+                                         this->get_solution(),
+                                         *ref_func,
+                                         cellwise_errors_u,
+                                         quadrature_formula,
+                                         VectorTools::L1_norm,
+                                         &comp_u);
+      VectorTools::integrate_difference (this->get_mapping(),this->get_dof_handler(),
+                                         this->get_solution(),
+                                         *ref_func,
+                                         cellwise_errors_p,
+                                         quadrature_formula,
+                                         VectorTools::L1_norm,
+                                         &comp_p);
+      VectorTools::integrate_difference (this->get_mapping(),this->get_dof_handler(),
+                                         this->get_solution(),
+                                         *ref_func,
+                                         cellwise_errors_ul2,
+                                         quadrature_formula,
+                                         VectorTools::L2_norm,
+                                         &comp_u);
+      VectorTools::integrate_difference (this->get_mapping(),this->get_dof_handler(),
+                                         this->get_solution(),
+                                         *ref_func,
+                                         cellwise_errors_pl2,
+                                         quadrature_formula,
+                                         VectorTools::L2_norm,
+                                         &comp_p);
+
+      const double u_l1 =  Utilities::MPI::sum(cellwise_errors_u.l1_norm(),this->get_mpi_communicator());
```

Comment:
can you please switch this to ``compute_global_error`` like I did in all other benchmarks in the last few weeks?

Reply:
this seems to have disappeared from what is now merged in the code